components/Category/Category.tsx

```diff
@@ -0,0 +1,12 @@
+import styles from "./Category.module.css";
+import { ReactNode } from "react";
+
+interface Props {
+  children: ReactNode;
+}
+
+const Category = ({ children }: Props) => {
```

Comment:
아마 화살표 함수로 만든 컴포넌트는 이렇게하면 children props type 명시안해줘도될거예요 :)
```suggestion
const Category: React.FC = ({ children }) => {
```